WcaOnRails/app/models/registration_payment.rb

```diff
@@ -6,4 +6,8 @@ class RegistrationPayment < ActiveRecord::Base
            as: "amount",
            allow_nil: true,
            with_model_currency: :currency_code
+
+  def amount_available_for_refund
+    amount_lowest_denomination + RegistrationPayment.where(registration_payment_id: id).sum("amount_lowest_denomination")
```

Comment:
I find this pretty confusing. I think my confusion is due to the name of this `registration_payment_id` column. My brain sees this as "We're looking for registration payments who have a registration payment id that is equal to this registration payment's id", which leads me in logic circles.

First idea that comes to mind: maybe rename `registration_payment_id` to `refund_for_registration_payment_id`? Or maybe `registration_payment_id_being_refunded`?